.github/workflows/on-pr.yml

```diff
@@ -0,0 +1,41 @@
+name: "Validate and plan"
```

Comment:
This isn't a good name for this workflow. This is a copy/paste from the terraform project I pulled the skeleton from.